app/components/step-footer.jsx

```diff
@@ -60,6 +62,13 @@ const useStylesFromThemeFunction = createUseStyles(theme => ({
   },
   next: {
     margin: '1rem 2rem',
+    width: '100%',                 // Flexible width based on parent
+    maxWidth: '13.8125rem',        // Ensures it doesn't exceed the design width
```

Comment:
Since we are using maxWidth, we also have to look at the mobile layout in figma. The mobile layout is a little thinner.   theme.condensedWidthBreakPoint can be used in a media query to decide.  Look at other modules that use condensedWidthBreakPoint for an example.

Also check the height  and other dimensions in mobile layout to make sure they are what they need to be.